scenes/enemy-commander/EnemyCommander.gd

```diff
@@ -0,0 +1,39 @@
+tool
+extends Node2D
+
+export(NodePath) var wave_manager_path
+var wave_manager
+export(NodePath) var call_button_path
+var call_button
+
+export(Array) var wave_sequence = [
+#	[time from prev. wave end to to next call button visible, time to wave spawn],
+	[2, 15],
+	[15, 30],
+]
+var wave_counter = 0
```

Comment:
We have a wave counter inside the Wave Manager already, do you think we can use that one?

Reply:
You're right, we can use the wave_counter var that is inside the Wave Manager.

Note:
- Replace `wave_counter` with `wave_manager.wave_counter + 1` in lines 27..29
- Delete lines 30 and 14 (in this order)